packages/dds/tree/src/test/util/testChangeRebaser.spec.ts

```diff
@@ -0,0 +1,85 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { ChangeRebaser } from "../../rebase";
+import { AnchorSet } from "../../tree";
+import { verifyChangeRebaser } from "../../util";
+
+function commutativeRebaser<TChange>(data: {
+    compose: (changes: TChange[]) => TChange;
+    invert: (changes: TChange) => TChange;
+    rebaseAnchors: (anchor: AnchorSet, over: TChange) => void;
+}): ChangeRebaser<TChange> {
+    return {
+        rebase: (change: TChange, over: TChange) => change,
+        ...data,
+    };
+}
+
+const counterRebaser = commutativeRebaser({
+    compose: (changes: number[]) => changes.reduce((a, b) => a + b, 0),
+    invert: (change: number) => -change,
+    rebaseAnchors: (anchor: AnchorSet, over: number) => {},
+});
```

Comment:
I changed my mind about trying to import this from the code Craig committed: it's a small amount of code and it's nice that the test will not become stale if we change the production code.

What would be nice though is to get rid of the `commutativeRebaser` since we're not reusing it. Something like this might work:
```suggestion
const counterRebaser: ChangeRebaser<number> = {
    compose: (changes: number[]) => changes.reduce((a, b) => a + b, 0),
    invert: (change: number) => -change,
    rebase: (change: number, over: number) => change,
    rebaseAnchors: (anchor: AnchorSet, over: number) => {},
});
```